2021-04-23 10:40:44,238 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:41:36,502 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:47:29,711 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:47:54,866 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:50:29,788 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:50:43,875 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:51:07,322 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:52:32,421 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:52:55,955 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:54:23,581 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 10:55:43,449 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 11:36:10,034 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 16:31:24,423 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 16:31:26,774 ERROR: Exception on / [GET] [in c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\samy.doreau\Dropbox\Code\flasky\app\routes.py", line 29, in index
    return render_template("index.html", title='Home Page', form=form,
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\jinja2\loaders.py", line 127, in load
    code = environment.compile(source, name, filename)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\jinja2\environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\samy.doreau\Dropbox\Code\flasky\app\templates\index.html", line 17, in template
    {% endiff %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endiff'. Jinja was looking for the following tags: 'elif' or 'else' or 'endif'. The innermost block that needs to be closed is 'if'.
2021-04-23 16:31:51,749 INFO: Microblog startup [in C:\Users\samy.doreau\Dropbox\Code\flasky\app\__init__.py:43]
2021-04-23 16:32:14,995 ERROR: Exception on /index [POST] [in c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\samy.doreau\dropbox\code\flasky\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\samy.doreau\Dropbox\Code\flasky\app\routes.py", line 23, in index
    post = Post(body=form.post.body, author=current_user)
NameError: name 'Post' is not defined
